exercises/simple-linked-list/example.go

```diff
@@ -0,0 +1,74 @@
+package linkedlist
+
+import "fmt"
+
+type Element struct {
+	data int
+	next *Element
+}
+
+type List struct {
+	head *Element
+	size int
+}
+
+func New(array []int) *List {
+	res := List{nil, 0}
+	for _, elem := range array {
+		res.Push(elem)
+	}
+	return &res
+}
+
+func (list *List) Size() int {
+	return list.size
+}
+
+func (list *List) Push(elem int) {
+	newNode := Element{elem, nil}
+	if list.head == nil {
+		list.head = &newNode
+	} else {
+		node := list.head
+		for node.next != nil {
+			node = node.next
+		}
+		node.next = &newNode
+	}
+	list.size++
+}
+
+func (list *List) Pop() error {
```

Comment:
might be useful to return popped value

Reply:
sure, I will change this and update the tests